recipes/ciocheck/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ciocheck" %}
+{% set version = "0.1.1" %}
+{% set hash_type = "sha1" %}
+{% set hash = "8cfedcc1d462f6f55ab33d2ea08b91ad55dca6e9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: v0.1.1
+  git_url: https://github.com/ContinuumIO/ciocheck.git
```

Comment:
I'm not sure what the policy is (if there even is one), but I feel like most packages either pull from pypi.io or the the github release archive (https://github.com/ContinuumIO/ciocheck/archive/v0.1.1.tar.gz).